src/components/Table/TableCell/TableCell.module.scss

```diff
@@ -17,6 +18,13 @@
     white-space: nowrap;
   }
 
+  &.sticky {
+    position: sticky;
+    left: 0;
+    font-size: var(--table-header-font-size);
+    font-weight: 600;
```

Comment:
This was to match the font-weight in the header. While the weight is appropriate for the use case, we don't have it in the design tokens. Should it be added? 
<img width="380" alt="Screen Shot 2020-10-26 at 3 03 03 PM" src="https://user-images.githubusercontent.com/5695990/97233693-9a89ef00-179c-11eb-9862-90670301723e.png">